src/Symfony/Bridge/PsrHttpMessage/Factory/HttpFoundationFactory.php

```diff
@@ -190,8 +190,6 @@ private function createCookie(string $cookie): Cookie
 
             if ('samesite' === strtolower($name) && null !== $value) {
                 $samesite = $value;
-
-                continue;
```

Comment:
I don't think this change is required. We might add another one block in the future after this one, and then this one will require continue statement again, but it might be missed by mistake. Now all blocks have continue, this is ok.

Reply:
I'm not really sold by the "missed by mistake" argument, but I like the consistency point. That's reverted for this file :+1: